WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/ShippingLabelFormViewController.swift

```diff
@@ -361,7 +369,7 @@ private extension ShippingLabelFormViewController {
             ServiceLocator.noticePresenter.enqueue(notice: notice)
             return
         }
-        let phoneNumberRequired = type == .origin && viewModel.customsFormRequired
+        let phoneNumberRequired = viewModel.customsFormRequired
```

Comment:
minor nit (feel free to left as is if it makes more sense to you without the `is`): 
```suggestion
        let isPhoneNumberRequired = viewModel.customsFormRequired
```